source/access/text_mode_access_using_putty.rst

```diff
@@ -3,11 +3,20 @@
 Text-mode access using PuTTY
 ============================
 
-Prerequisite: PuTTY and WinSCP
-------------------------------
+Prerequisite
+------------
+
+.. tab-set::
+
+   .. tab-item:: KU Leuven
 
-You've :ref:`generated a public/private key pair with PuTTY
-<generating keys putty>` and have an approved account on the VSC clusters.
+      You have an approved :ref:`VSC account <access>`.
+
+   .. tab-item:: UHasselt, UGent, VUB, UAntwerpen
+
+      You have already
+      :ref:`generated a public/private key pair with PuTTY <generating keys putty>`,
+      and have an approved :ref:`VSC account <access>` on the VSC clusters.
 
```

Comment:
I suppose the tabs refer to the site where the target cluster is hosted, right? If so, then it may be good to mention this explicitly (otherwise KU Leuven-affiliated users might think that they don't need a key pair for any VSC cluster, which isn't true). And if so then there's also no need for an `UHasselt` tab.

Reply:
indeed. Adding
"You have an approved VSC account, for accessing non-KU Leuven clusters you also need a public/private key pair with PuTTY".

Reply:
You can need to do KU Leuven/UHasselt, instad of UHasselt in the other tab. The information is more for the infrastrcuture of the site and not for the users.

Reply:
Sure. fixed in 4ab847e